site/source/docs/contributing/developers_guide.rst

```diff
@@ -95,6 +98,46 @@ automatically when you create a pull request, and they should all pass. If you
 run into trouble with a test failure you can't fix, please let the developers
 know.
 
+Bisecting
+=========
+
+If you find a regression, bisection is often the fastest way to figure out what
+went wrong. This is true not just for finding an actual regression in Emscripten
+that you want to fix, but also if your project stopped working when you upgrade,
+and you need to investigate if it's an Emscripten regression or something else.
+
+The normal ``git bisect`` workflow can work if you just need to bisect a single
+repository. For example, you can bisect only on the emscripten repo if you are on
+a range that all works with the same versions of LLVM and Binaryen (which was
+mentioned earlier, are two separate codebases that are depended on).
+
+If you have a large bisection range, you generally can't bisect a single repo.
+You can still bisect, though! To do that you need the emsdk and to understand
+how the
+`release process <https://github.com/emscripten-core/emscripten/blob/master/docs/process.md#release-processes>`_
+works for all the repos together. The key "trick" is that
+
+::
```

Comment:
I think the `::` can be attached to the end of the paragraph.  I've been meaning to go though and change this everything since its less veritical whitespace and also the `::` then reads logically as `:`  and avoids `:` followed by `::`.